framework/include/model.hpp

```diff
@@ -566,23 +566,23 @@ namespace gvk
 	 *	but it should help to express the intent better. Use it with `cgb::append_vertex_data_and_indices`!
 	 */
 	template <class... Types>
-	_NODISCARD constexpr std::tuple<Types&&...> additional_vertex_data(Types&&... aArgs) noexcept {
+	[[nodiscard]] constexpr std::tuple<Types&&...> additional_vertex_data(Types&&... aArgs) noexcept {
```

Comment:
That's cool. Thanks.

Reply:
np, it was necessary for compilers other than MSVC anyways